app/controllers/admin/userManagement.py

```diff
@@ -16,14 +16,20 @@ def manageUsers():
     user = eventData['user']
     method = eventData['method']
     username = re.sub("[()]","", (user.split())[-1])
-    user = User.get_by_id(username)
+
+    try:
+        user = User.get_by_id(username)
+    except Exception as e:
+        print(e)
+        flash(username + " is an invalid user.", "danger")
+        return ("danger")
```

Comment:
If it's an error, return an error code. Like `return ("danger"), 500`

Reply:
resolved